.eslintrc.js

```diff
@@ -35,8 +33,48 @@ module.exports = {
     'react/forbid-prop-types': 'off',
     'react/prefer-stateless-function': 'off',
     'react/prop-types': 'off',
+    'react/jsx-filename-extension': ['error', { extensions: ['.tsx', '.jsx'] }],
   },
   overrides: [
+    {
```

Comment:
Medium term, we should consolidate these and apply the same rules everywhere, but there are many issues that fail in JS files, so for now this was required to get linting to work.

Reply:
SGTM!